server/lib/queries.js

```diff
@@ -183,7 +192,7 @@ export const deleteResourceServer = req =>
     });
 
 const getGrantId = req =>
-  makeRequest(req, 'client-grants', 'GET')
+  makeRequest(req, 'client-grants', 'GET', { client_id: config('AUTH0_CLIENT_ID'), audience: 'urn:auth0-authz-api' })
```

Comment:
Pass the options supported in https://auth0.com/docs/api/management/v2#!/Client_Grants/get_client_grants to filter instead of getting all results.

Reply:
Are we fine not doing pagination here?

Reply:
I believe so, because this will only return 1 or 0 results (depending on if there's a grant for the client_id/API combination or not).